ec/ecresource/deploymentresource/create.go

```diff
@@ -95,9 +96,13 @@ func (r *Resource) Create(ctx context.Context, req resource.CreateRequest, resp
 		filters = request.Settings.TrafficFilterSettings.Rulesets
 	}
 
-	deployment, diags := r.read(ctx, *res.ID, nil, &plan, res.Resources, filters)
+	var migrateTemplateRequest *deployments.MigrateDeploymentTemplateOK
+	deployment, diags := r.read(ctx, *res.ID, nil, &plan, res.Resources, filters, migrateTemplateRequest)
 	updatePrivateStateTrafficFilters(ctx, resp.Private, filters)
 
+	// Store migrate request in private state
+	updatePrivateStateMigrateTemplateRequest(ctx, resp.Private, migrateTemplateRequest)
```

Comment:
`migrateTemplateRequest` is alwyas `nil` here too. I think we can remove the line too.

Reply:
Addressed in https://github.com/elastic/terraform-provider-ec/pull/755/commits/4c70d0034521bc6e2be803e5472948612fb64f2c.